engine/apps/public_api/views/incidents.py

```diff
@@ -54,6 +56,20 @@ def get_queryset(self):
             queryset = queryset.filter(channel_filter__public_primary_key=route_id)
         if integration_id:
             queryset = queryset.filter(channel__public_primary_key=integration_id)
+        if state:
+            choices = dict(AlertGroup.STATUS_CHOICES)
+            choice = [i for i in choices if choices[i] == state.lower().capitalize()]
```

Comment:
Probably a good idea to add a suggestion on available state options, in case a provided `state` is not one.
Something similar to this:

```suggestion
            try:
                choice = [i for i in choices if choices[i] == state.lower().capitalize()][0]
            except KeyError:
                valid_choices_text = ", ".join([status_choice[1].lower() for status_choice in AlertGroup.STATUS_CHOICES])
                raise BadRequest(detail={"state": f"Must be one of the following: {valid_choices_text}"})
```